addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -732,7 +745,7 @@ def _import_fattura_pa_line(self, element, invoice_line_form, extra_info=None):
                         percentage = round(tax_amount / price_subtotal * 100)
 
         natura_element = element.xpath('.//Natura')
-        invoice_line_form.tax_ids = []
+        invoice_line_form.tax_ids = ()
```

Comment:
Why ? This diff seems useless

Reply:
The current line does nothing. That is it doesn't "clean" the `tax_ids`.
This change thus allows the "clean" to be done as intended.
(iirc this is a type thing but I can't say for sure as I haven't really dug into it.)

Reply:
Using the empty set command is way more explicit
```
        invoice_line_form.tax_ids = [Command.set([])]
```

Relying on the type of the other able should be avoided as the behavior is probably undefined